### DISCUSSION TITLE
Chapter 48 links on RESTEasy v6.2.7.Final documentation do not work

### DISCUSSION THREAD

**Discussion by jdomingo23** [resteasy/resteasy-examples#191] [Q&A]
Hi RESTEasy Team!

I am currently in the early stages of migrating some old Spring REST applications from the old v3.x RESTEasy framework to the v6.2.7.Final.  I was reading the Spring Integration Chapter (48) of the RESTEasy v6.2.7.Final documentation, and I noticed the following links do not resolve to anything in the MAIN branch of the GIT repo:

1. The resteasy-spring-basic (Basic Examples URL anchor in Chapter 48.2)
2. The resteasy-spring-customized (Chapter 48.3).
3. The resteasy-spring-mvc (Chapter 48.4)

However, the resteasy-spring-boot option resolves fine.  Do you guys have an idea when these example projects will be made available for us to read through in the resteasy-examples project?  

Thanks much!

Cheers!
John

**Comment by jdomingo23** [comment]
Hi RESTEasy Team!

I was actually able to find the examples I was looking for under the "resteasy" project root (e.g., https://github.com/resteasy/resteasy-spring-examples/tree/main/resteasy-spring-basic).  So...the links on the Chapter 48 page of the v6.2.7.Final documentation just need to get updated accordingly.

Thanks!

John.

**Reply by jamezp** [reply]
Yes, sorry. Maybe we should add a link to the README with the Spring examples too.
